plugins/modules/dedicated_server_boot.py

```diff
@@ -68,7 +68,7 @@ def run_module():
     force_reboot = module.params['force_reboot']
     changed = False
 
-    bootid = {'harddisk': 1, 'rescue': 1122, 'rescue-customer': 46371}
+    bootid = {'harddisk': 1, 'rescue-customer': 46371, 'ipxe-shell': 203323, 'poweroff': 95083}
     if module.check_mode:
         module.exit_json(
             msg="{} is now set to boot on {}. Reboot in progress... - (dry run mode)".format(service_name, boot),
```

Comment:
For consistency in the file and the modules, can you  use fstring in the three `exit_json` calls?

Reply:
changed to fstring